packages/manager/src/features/Backups/AutoEnroll.tsx

```diff
@@ -65,14 +38,38 @@ export const AutoEnroll: React.FC<CombinedProps> = (props) => {
                 Backups pricing page
               </Link>
             </Typography>
-          </div>
+          </StyledDiv>
         }
-        className={classes.toggleLabel}
       />
-    </Paper>
+    </StyledPaper>
   );
 };
 
-const styled = withStyles(styles);
+const StyledPaper = styled(Paper, {
+  label: 'StyledPaper',
+})(({ theme }) => ({
+  backgroundColor: theme.bg.offWhite,
+  padding: theme.spacing(1),
+}));
+
+const StyledFormControlLabel = styled(FormControlLabel, {
+  label: 'StyledFormControlLabel',
+})(({ theme }) => ({
+  alignItems: 'flex-start',
+  display: 'flex',
+  marginBottom: theme.spacing(1),
+  marginLeft: 0,
+}));
+
+const StyledDiv = styled('div', {
+  label: 'StyledDiv',
+})(({ theme }) => ({
+  marginTop: theme.spacing(1) + theme.spacing(0.5),
```

Comment:
`theme.spacing()` returns a `string` so this code would result in `marginTop: 8px4px`. Do we want to change this to

```suggestion
  marginTop: theme.spacing(1.5),
```